core/src/main/java/io/grpc/inprocess/InProcessServerBuilder.java

```diff
@@ -28,6 +28,47 @@
  * its name.
  *
  * <p>The server is intended to be fully-featured, high performance, and useful in testing.
+ *
+ * <h3>Using JUnit TestRule</h3>
+ * The class "GrpcServerRule" (from the "grpc-java/testing") is a JUnit TestRule that
+ * creates a {@link InProcessServer} and a {@link io.grpc.ManagedChannel ManagedChannel}. This
+ * test rule contains the boiler plate code shown below. The classes "HelloWorldServerTest" and
+ * "HelloWorldClientTest" (from "grpc-java/examples") demonstrate basic usage.
+ *
+ * <h3>Usage example</h3>
+ * <h4>Server and client channel setup</h4>
+ * <pre>
+ *   MutableHandlerRegistry serviceRegistry = new MutableHandlerRegistry();
+ *   addServicesHelper(serviceRegistry); // your code here
```

Comment:
Move this line some point later? As it is it doesn't need the `MutableHandlerRegistry`.